angular-spa-ui/js/search-app/manage-account-module/manage-account-cfg.js

```diff
@@ -9,7 +9,8 @@ module.exports = function($stateProvider) {
             views: {
                 'module-content': {
                     templateUrl: '/accountModulePageTemplate.html',
-                    controller: 'accountController'
+                    controller: 'accountController',
+                    controllerAs: 'accountCtrl'
```

Comment:
controllerAs is not an obvious name of variable... Add additional letter "s" at the end of name of your variable. You will see why name "controllerAs" is not cool.